coreUtilities/build.gradle

```diff
@@ -59,6 +60,23 @@ dependencies {
     testFixturesImplementation group: 'org.slf4j', name: 'slf4j-api'
 }
 
+// Utility task to allow copying required libraries into a 'dependencies' folder for security scanning
+tasks.register('copyDependencies', Sync) {
```

Comment:
Lets also see about making this a shared task with all projects

Reply:
Makes sense; mind if I do it on the next pass?  I was hoping to push out some changes to the CodePipeline soon since it takes so long to finish.

Reply:
I just merged a new project for transformations, is that correct with our standards?

Reply:
It's not currently being scanned for SonarQube anyways (each project requires manual inclusion), so how about I tackle both including that new sub-project and the other changes suggested in the next PR?